src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/TokenBasedFormattingRule.cs

```diff
@@ -269,6 +272,10 @@ private static SyntaxList<UsingDirectiveSyntax> GetUsings(SyntaxNode node)
                     return CreateAdjustSpacesOperation(1, AdjustSpacesOption.ForceSpacesIfOnSingleLine);
                 }
 
+                // if it's a standalone empty statement, don't adjust it at all.
+                if (currentToken.Parent is EmptyStatementSyntax)
+                    return null;
```

Comment:
💭 It seems like the desired behavior here would be to have a preceding space except in the case the token is on a line by itself.

Reply:
sure. i can try that.